prow/entrypoint/run.go

```diff
@@ -33,13 +34,23 @@ import (
 )
 
 const (
+	// internalCode is greater than 256 to signify entrypoint
+	// chose the code rather than the command it ran
+	// http://tldp.org/LDP/abs/html/exitcodes.html
+	//
+	// TODO(fejta): consider making all entrypoint-chosen codes internal
+	internalCode = 1000
```

Comment:
I think this should be used for the two fields below this in addition to `PreviousErrorCode`.

Reply:
I agree in principle but would rather handle that on its own, later, hence the TODO. SGTY?

Don't want to change the behavior of existing entrypoint jobs right now